src/Casino/Roulette.tsx

```diff
@@ -40,13 +40,13 @@ const strategies: {
 } = {
   Red: {
     match: (n: number): boolean => {
-      if (n === 0) return false;
```

Comment:
this wasn't required, as redNumbers is an array that doesn't contain a zero, so would never match anyway